src/lib.rs

```diff
@@ -120,6 +120,39 @@ pub enum ImageType {
     Webp,
 }
 
+impl ImageType {
+    /// Calls the correct image size method based on the image type
+    ///
+    /// # Arguments
+    /// * `reader` - A reader for the data
+    pub fn reader_size<R: BufRead + Seek>(&self, reader: &mut R) -> ImageResult<ImageSize> {
+        match self {
+            ImageType::Aseprite => aesprite::size(reader),
```

Comment:
Can you please add back the cfg flags for all of these? Otherwise feature flags won't work.